lib/gtag.js

```diff
@@ -0,0 +1,29 @@
+export const GA_TRACKING_ID = process.env.GA_TRACKING_ID;
```

Comment:
This ought to also be 

```suggestion
export const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GA_TRACKING_ID;
```

The NEXT_PUBLIC is required by next js to have the env be exposed to the application

https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser